### PR TITLE
Fix for #3777 - use clearer message if processing plugin in missing

### PR DESCRIPTION
**Pull request by timlinux**

### CODE REVIEW

File: safe/impact_function/impact_function.py [hunk 1]
```diff
@@ -1021,9 +1021,16 @@ def run(self):
             suggestion_heading = m.Heading(
                 tr('Suggestion'), **SUGGESTION_STYLE)
             suggestion = tr(
-                'Check in your .qgis2/python/plugins directory that you do '
-                'not have a processing folder. You should use the Processing '
-                'plugin provided by QGIS.')
+                'InaSAFE depends on the QGIS Processing plugin. This is a '
+                'core plugin that ships with QGIS. It is also possible to '
+                'install the processing plugin from the QGIS Plugin Manager, '
```
Comment (Gustry) [inline safe/impact_function/impact_function.py]: The plugin is not shipped anymore on plugins.qgis.org. You can't download it. You can only enable/disable it in the plugin manager.

**Comment by Gustry** [comment]
Looks good to me, thanks !

**Comment by timlinux** [comment]
@gustry I guess you also need to go back to your review to approve the PR:

screen shot 2017-02-08 at 2 26 35 pm

**Comment by Gustry** [comment]
Ah yes sorry. You could stil merge

**Pull request merged**